### ISSUE TITLE
Viewable stuttering doing anything on the emulator

### ISSUE THREAD

**Issue by RobertMVelasquez1996** [ibrumby/Crossrunner-Issues#26]
Hey there.
I am having a weird bug in my download of Crossrunner. When I try to do anything such as open or close a directory or disk to select something else, the emulator chugs badly, as if it is running on its last legs in how slow it is. It was not this way when I first downloaded it last week. The open and close animations go at least, to me, 10 frames per second or slower, when it should definitely be faster.

Maybe a bug fix could be done to fix this. Just letting you know to maybe check your code to see if something could be optimized in how the emulator runs.

Thank you in advance for reading this.
Hopefully something gets done about this slowdown of the emulator.

**Comment by ibrumby** [comment]
When you say open or close a directory - do you mean in Finder?

**Comment by RobertMVelasquez1996** [comment]
Yes. That is the only bug I am seeing in my download of the software, the other parts of the emulator work just fine. Somehow even better, especially because your software is the only one that supports HFS disk images, whereas the other IIGS emulators do not even know what to do with it so they don't show it as a usable disk image.
Edit 1: if it helps to understand what is going on, the slowdown even occurs in the unlimited speed setting in the program.
